src/components/MuiVirtualizedTable/ColumnHeader.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React from 'react';
+import clsx from 'clsx';
+import { makeStyles } from '@mui/styles';
+
+import FilterAltOutlinedIcon from '@mui/icons-material/FilterAltOutlined';
+
+import ArrowUpwardIcon from '@mui/icons-material/ArrowUpward';
+import ArrowDownwardIcon from '@mui/icons-material/ArrowDownward';
+
+const useStyles = makeStyles((theme) => ({
+    label: {
+        fontWeight: 'bold',
+        padding: '16px',
+    },
+    divFlex: {
+        display: 'flex',
+        flexDirection: 'row',
+        alignItems: 'center',
+    },
+    divNum: {
+        display: 'flex',
+        flexDirection: 'row-reverse',
+        alignItems: 'center',
+    },
+    sortDiv: {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+    },
+    sortButton: {
+        fill: 'currentcolor',
+    },
+    filterButton: {
+        stroke: 'currentcolor',
+    },
+    filterTooLossy: {
+        stroke: theme.palette.secondary.main,
+    },
+    transparent: {
+        opacity: 0,
+    },
+    hovered: {
+        opacity: 0.5,
+    },
+}));
+
+// Shows an arrow pointing to smaller value when sorting is active.
+// signedRank of 0 means no sorting, we only show the arrow on hovering of the header,
+// in the same direction as it will get if clicked (once).
+// signedRank > 0 means sorted by ascending value from lower indices to higher indices
+// so lesser values are at top, so the upward arrow
+const SortButton = (props) => {
+    const classes = useStyles();
+    const sortRank = Math.abs(props.signedRank);
+    const visibilityClass =
+        !props.signedRank &&
+        (props.headerHovered ? classes.hovered : classes.transparent);
+    return (
+        <div className={clsx(classes.sortDiv)} onClick={props.onClick}>
```

Comment:
Why a div and not a IconButton from MUI ?
https://mui.com/material-ui/api/icon-button/